tests/app/celery/test_process_delivery_status_result_tasks.py

```diff
@@ -536,3 +536,28 @@ def test_get_include_payload_status_exception(notify_api, mocker, sample_notific
         'app.celery.process_delivery_status_result_tasks.dao_get_callback_include_payload_status', side_effect=exception
     )
     assert not _get_include_payload_status(sample_notification())
+
+
+@pytest.mark.serial
+def test_process_delivery_status_redacts_personalisation(
+    mocker,
+    sample_delivery_status_result_message,
+    sample_template,
+    sample_notification,
+):
+    """
+    Test that the Celery task will redact personalisation if the delivery status is in a final state.
+    """
+    sample_notification(
+        template=sample_template(content='Test ((foo))'),
+        reference='SMyyy',
```

Comment:
Hard coding values can interfere with parallel test execution.
```suggestion
        reference=str(uuid4()),
```
You did this below, in a different test.

Reply:
These reference values are baked into the encoded fixture, and the tests are marked serial, unfortunately.